docs/operations.md

```diff
@@ -66,6 +66,10 @@ Below is a high-level overview of how the communication flow operates:
 2. Upon receipt of the request, `wfx` sets the `Content-Type` header to `text/event-stream`.
 3. The server then initiates a stream of job events in the response body, allowing clients to receive instant updates.
 
+**Note**: To prevent the connection from being closed due to inactivity (e.g., when no job events occur), periodic keep-alive events are sent during such idle periods.
+This ensures the connection remains open, avoiding closure by, e.g., proxies or the kernel.
```

Comment:
This is due since one may not be able to control all involved parties (clients, wfx, and intermediates). If this was the case, then keep alive messages would be not needed. Is it worth to add this reasoning?